scripts/migrations/2024_07_change_main_contact.pl

```diff
@@ -54,22 +54,24 @@
 		)    # shouldn't be a main contact
 		)
 	{
+		print STDERR "previous main contact: " . $org_ref->{main_contact} . "\n";
```

Comment:
Probably not an issue: The previous `print` statements just output to `STDOUT` instead of `STDERR`. Is that intentional?

Reply:
I'll change it, thanks